inits.json

```diff
@@ -0,0 +1,54 @@
+{
+    "user_info": [
+        {
+            "Directory with FITS files": "./sample-data/HatP32Dec202017",
+            "Directory to Save Plots": "./sample-data",
+            "Directory of Flats": "n",
```

Comment:
I'm a little worried about having the answer be "n" here. Maybe we should make the code more robust to accept a "n/a" or "none" or give the user instructions for what to do if they do not have cals? Can we add a comment in-line after the `,`? I.e., can we put in `# please put 'n' if no calibrations` - I'm not sure if this will work with the json file format or not

Reply:
^ definitely address this comment

Reply:
Looks like JSONs do not support comments, but we can get around that by doing something like this: https://stackoverflow.com/questions/244777/can-comments-be-used-in-json